src/factuality_eval/prompt_utils.py

```diff
@@ -0,0 +1,83 @@
+"""Utilities for loading and formatting prompts. Adapted from LettuceDetect."""
+
+from __future__ import annotations
+
+from pathlib import Path
+from string import Template
+
+# Type for supported languages
+Lang = str  # "en", "de", "fr", "es", "it", "pl", "cn"
+
+LANG_TO_PASSAGE = {
+    "da": "afsnit",
+    "en": "passage",
+    "de": "Passage",
+    "fr": "passage",
+    "es": "pasaje",
+    "it": "brano",
+    "pl": "fragment",
+    "cn": "段落",
+    "hu": "szövegrészlet",
+}
+
+# Full language names for each language code
+LANG_TO_FULL_NAME = {
+    "da": "Danish",
+    "en": "English",
+    "de": "German",
+    "fr": "French",
+    "es": "Spanish",
+    "it": "Italian",
+    "pl": "Polish",
+    "cn": "Chinese",
+    "hu": "Hungarian",
+}
+
+PROMPT_DIR = Path(__file__).parent.parent / "prompts"
+
+
+class PromptUtils:
+    """Utility class for loading and formatting prompts."""
+
+    @staticmethod
+    def load_prompt(filename: str) -> Template:
+        """Load a prompt template from the prompts directory.
+
+        :param filename: Name of the prompt file
+        :return: Template object for the prompt
+        :raises FileNotFoundError: If the prompt file doesn't exist
+        """
+        path = PROMPT_DIR / filename
+        if not path.exists():
+            raise FileNotFoundError(f"Prompt file not found: {path}")
+        return Template(path.read_text(encoding="utf-8"))
+
+    @staticmethod
+    def format_context(context: list[str], question: str | None, lang: Lang) -> str:
+        """Format context and question into a prompt.
+
+        :param context: List of passages
+        :param question: Question (None for summarization tasks)
+        :param lang: Language code
+        :return: Formatted prompt
```

Comment:
Converting to Google format:

```suggestion
        Args:
            context:
                List of passages.
            question:
                The question, or None for summarization tasks.
            lang:
                The language code.
                
        Returns:
            Formatted prompt.
```